docs/topics/mta-cli-args.adoc

```diff
@@ -187,27 +213,4 @@ A space-delimited list of one or more target technologies, servers, platforms, o
 ----
 
 
-The `--target` argument now provides version support, which follows the link:http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html[Maven version range syntax]. This instructs {ProductShortName} to only run the rulesets matching the specified versions. For example, `--target eap7`.
-
-[WARNING]
-====
-When migrating to JBoss EAP, be sure to specify the version in the target, for example, `eap6`. Specifying only `eap` will run rulesets for all versions of JBoss EAP, including those not relevant to your migration path.
-
-See link:{ProductDocIntroToMTAGuideURL}/index#migration_paths_getting-started-guide[Supported migration paths] in _{IntroToMTABookName}_ for the appropriate JBoss EAP version.
-====
-
-////
-[id="cli-packages-argument_{context}"]
```

Comment:
@istein1 -  section on *Packages* removed here 
This only leaves three references of `--packages` to resolve for [MTA-4220](https://issues.redhat.com/browse/MTA-4220)